src/detect-transform-base64.c

```diff
@@ -45,7 +44,7 @@ static void DetectTransformFromBase64DecodeRegisterTests(void);
 #endif
 static void TransformFromBase64Decode(InspectionBuffer *buffer, void *options);
 
-#define DETECT_TRANSFORM_FROM_BASE64_MODE_DEFAULT (uint8_t) Base64ModeRFC4648
+#define DETECT_TRANSFORM_FROM_BASE64_MODE_DEFAULT (uint8_t) RFC4648
```

Comment:
can we have a more descriptive name for RFC4648? Just the RFC in isolation is a bit too terse

Reply:
Ok. I'll switch it back. I did to because on the Rust side it looked redundant but I agree on the C side, it looks incomplete